### ISSUE TITLE
Emergency Priority (2) not working as expected

### ISSUE THREAD

**Issue by soakes** [jryberg/notify-via-pushover#1]
When trying to use 'Emergency Priority' for notifications, you get the following error message.

<code>{"expire":"must be supplied with priority=2","errors":["expire must be supplied with priority=2"],"status":0,"request":"7ecdec9303142ec6887d14dcf1f78609"}</code>

This is because your missing two parameters in the script which sets expire and retry values/options.

**Comment by jryberg** [comment]
Sorry about the delay. I will take a look at this issue

**Comment by soakes** [comment]
No worries.  This was just to inform you of the problems which I found.  

As its just a bash script, I fixed it myself for my own uses.

If you would like to see what ive done to fix this, ive uploaded a copy on dropbox as I dont have time right now to sort it onto github.  Just diff the script over your version to see the changes.

https://www.dropbox.com/s/r8qp0o4j9zfqhxy/notify-via-pushover?dl=0

Hope this helps.

Simon
